go.mod

```diff
@@ -9,7 +10,7 @@ require (
 	github.com/gammazero/deque v0.0.0-20180920172122-f6adf94963e4 // indirect
 	github.com/gammazero/workerpool v0.0.0-20181230203049-86a96b5d5d92
 	github.com/golangci/golangci-lint v1.40.1
-	github.com/grpc-ecosystem/go-grpc-middleware v1.3.0 // indirect
+	github.com/grpc-ecosystem/go-grpc-middleware v1.3.0
```

Comment:
Are the changes to this file needed for your change? If not it may be best to leave it as was.

Reply:
Not intended, reverted in latest commit